src/buildfile

```diff
@@ -61,10 +64,10 @@ hxx{*}: install = false
 if $version.pre_release
 {
   lib{benchmark}: bin.lib.version = @"-$version.project_id"
-  lib{benchmark_main}: bin.lib.version = @"-$version.project_id"
+  liba{benchmark_main}: bin.lib.version = @"-$version.project_id"
```

Comment:
Library version is meaningless for a static library so I would drop it.

Reply:
Done